### ISSUE TITLE
P2.py

### ISSUE THREAD

**Issue by benefoon** [benefoon/Credit-Risk-Prediction-System-Using-Machine-Learning#2]
from sklearn.linear_model import LogisticRegression
from sklearn.metrics import accuracy_score, classification_report

# Initialize and train the model
model = LogisticRegression()
model.fit(X_train, y_train)

# Make predictions
y_pred = model.predict(X_test)

# Evaluate the model
print("Accuracy:", accuracy_score(y_test, y_pred))
print(classification_report(y_test, y_pred))
